wgpu-hal/src/dx11/adapter.rs

```diff
@@ -129,7 +130,8 @@ impl super::Adapter {
         }
 
         if feature_level >= FL11_0 {
-            downlevel |= wgt::DownlevelFlags::INDIRECT_EXECUTION;
+            downlevel |= wgt::DownlevelFlags::INDIRECT_EXECUTION
+                | wgt::DownlevelFlags::WEBGPU_TEXTURE_FORMAT_SUPPORT;
```

Comment:
```suggestion
            downlevel |= wgt::DownlevelFlags::INDIRECT_EXECUTION;
            downlevel |= wgt::DownlevelFlags::WEBGPU_TEXTURE_FORMAT_SUPPORT;
```
nit to bring in line with rest of the code.